### PR TITLE
feat:managedTeams added header, paragraph, positioned border, and cha…

### PR DESCRIPTION
**Pull request by ell-sie**
# Description
Added content and styling for the African Team Box section.

## Motivation and Context
- [x] Added content: The added content aims to provide information about the African Team Box and its benefits.
- [x] Styling changes: Styling updates have been made to enhance the visual presentation and user experience.

## How Has This Been Tested, and How are your changes run?
- [x] Local testing: The changes have been tested locally by adding the content and reviewing its appearance.
- [x] Code execution: The code is run using appropriate build tools.
- [x] Impact assessment: The impact of the changes on other parts of the code has been assessed to prevent unintended side effects.

## Screenshots (if appropriate):
image

### CODE REVIEW

File: src/pages/managedTeams.html
Comment (peninah98) [inline src/pages/managedTeams.html:1]: Well done!

**Review by peninah98 — APPROVED** [review]

**Review by peninah98 — APPROVED** [review]

**Comment by alainmigabo** [comment]
Please delete the dist folder to avoid conflicts in dev branch

**Comment by ell-sie** [comment]
> Please delete the dist folder to avoid conflicts in dev branch

I delete it.

**Review by Nezerwa — APPROVED** [review]
congratulations your task is complete it's time to merge!

**Pull request merged**